analysis/PE_AMI cox model code stata.do

```diff
@@ -0,0 +1,91 @@
+* Script to run PE and ATE outcomes in the electively unvaccinated population in STATA
+* Electively unvaccinated is a whole population sample
+
+clear
+cd "C:\Users\rd16568\OneDrive - University of Bristol\Documents\CONVALESCENCE\CVD stata analysis"
```

Comment:
Remove this line. The working directory is automatically set to 'post-covid-vaccinated' when this code runs in the real data.